toolbox

```diff
@@ -70,6 +70,7 @@ toolbox_image=""
 toolbox_runtime_directory="$XDG_RUNTIME_DIR"/toolbox
 user_id_real=$(id -ru 2>&3)
 verbose=false
+podman="podman"
```

Comment:
While trying to rebase this branch before merging, I realized that it would be handy if the name of the variable was slightly different from just `podman`. eg., if the variable is `podman_command`, then one can quickly grep for `podman ` to locate direct invocations left unconverted in the code.